source/beamline/BioXAS/BioXASSSRLMonochromatorEnergyControl.cpp

```diff
@@ -0,0 +1,23 @@
+#include "BioXASSSRLMonochromatorEnergyControl.h"
+
+BioXASSSRLMonochromatorEnergyControl::BioXASSSRLMonochromatorEnergyControl(const QString &name, const QString &readPVname, const QString &writePVname, const QString &movingPVname, const QString &stopPVname, QObject *parent, double tolerance, double moveStartTimeoutSeconds, AMAbstractControlStatusChecker *statusChecker, int stopValue, const QString &description) :
+    AMPVwStatusControl(name, readPVname, writePVname, movingPVname, stopPVname, parent, tolerance, moveStartTimeoutSeconds, statusChecker, stopValue, description)
+{
+
+}
+
+BioXASSSRLMonochromatorEnergyControl::~BioXASSSRLMonochromatorEnergyControl()
+{
+
+}
+
+AMControl::FailureExplanation BioXASSSRLMonochromatorEnergyControl::move(double setpoint)
```

Comment:
That's a good question @dretrex 
@helfrij this is one of those things that crops up a lot on actual controls and can cause no end of headaches. What behavior are you trying to address with this check? Maybe the three of us should meet up and discuss it.